### PR TITLE
Banana Float with Animated Eyes

### PR DESCRIPTION
**Pull request by golneshin**
This a ZTM project which is a challenge in Hacktoberfest. The challenge asked for at least one animation, then I created this project that included three animations.

### CODE REVIEW

**Comment by LaurelineP** [comment]
Hi there, congrats and thanks for the contribution 🚀 
Your contribution is missing the `include.js` modification, could you do it please? 
If any questions, this is explained on the README.md file :)

File: Art/golneshin_mardomak/mardomak.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Mardomak</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+  <script src='main.js'></script>
```
Comment (LaurelineP) [inline Art/golneshin_mardomak/mardomak.html:9]: # Feedback - no script tag in html file
Please remove the line with the script tag

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Please also provide the modifications for the `incude.js` file

**Comment by l-white** [comment]
Closing pull request due to inactivity.

**Pull request closed without merging**